deployments/liqo/subcharts/authService/templates/auth-service-deploy.yaml

```diff
@@ -140,6 +140,7 @@ spec:
         run: auth-service
     spec:
       serviceAccountName: auth-service-sa
+      {{ if not .Values.ingress.enable }}
       initContainers:
         - name: cert-creator
           image: nginx
```

Comment:
I know this is not part of this PR, but I would avoid using an image without a tag, as it forces the download and increases the start-up time